### PR TITLE
Update 3.0.2

### PR DESCRIPTION
**Pull request by SC426**
Jira ticket: https://anaconda.atlassian.net/browse/PKG-578
Upstream repo: https://github.com/pallets-eco/flask-sqlalchemy
Changelog: https://github.com/pallets-eco/flask-sqlalchemy/blob/main/CHANGES.rst

### CODE REVIEW

**Review by cbouss — APPROVED** [review]
LGTM, please remove abs.yaml too

**Review by ELundby45 — APPROVED** [review]
LGTM.

**Review by varlackc — COMMENTED** [review]
do we still want to keep the `abs.yaml` file?

https://github.com/AnacondaRecipes/flask-sqlalchemy-feedstock/blob/758bff3d2f254bd297678123a6a1213f9a9c72b0/abs.yaml

We want to be careful with the pinnings on the `host` section.
We want to avoid using comparison operators in the `host` section if possible.

https://github.com/AnacondaRecipes/flask-sqlalchemy-feedstock/blob/758bff3d2f254bd297678123a6a1213f9a9c72b0/recipe/meta.yaml#L22

File: recipe/meta.yaml [hunk 1]
```diff
@@ -8,32 +8,43 @@ package:
...
+  skip: True  # [py<37]
   number: 0
-  noarch: python
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
     - python
     - pip
+    - pdm-pep517 >=1.0.0
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:22]: As we have **v1.0.6** you can remove pinning `>=1.0.0` from host

**Pull request merged**